### PR TITLE
add get_issue_links

### PR DESCRIPTION
**Pull request by mjtice**

### CODE REVIEW

**Review by arm4b — APPROVED** [review]
Thanks for the contribution!
Looks good to me 👍 

Just let us know if it's ready to be merged.

**Comment by mjtice** [comment]
Thanks, it's ready whenever you are.

File: actions/get_issue_links.py
Comment (floatingstatic) [inline actions/get_issue_links.py]: This could be condensed to:

```
result = [to_links_dict(i) for i in issue.fields.issuelinks]
```

File: actions/lib/formatters.py
Comment (floatingstatic) [inline actions/lib/formatters.py]: Similar comment here, we can use a list comprehension:

```
result['links'] = [to_links_dict(i) for i in issue.fields.issuelinks]
```

File: actions/lib/formatters.py
Comment (floatingstatic) [inline actions/lib/formatters.py]: Minor nit, you can probably just `return` direct here instead of assigning to `result`. But fine if we want to leave that as-is.

**Review by floatingstatic — APPROVED** [review]
Sorry for the delay getting to this. Looks good to me. Tested in my st2-docker setup and works just fine. Left a couple of style comments, none of them blocking a merge here. If you are done making changes or do not want to address those minor comments we can merge this today. Let me know.

File: actions/lib/formatters.py
Comment (mjtice) [reply actions/lib/formatters.py]: I can update. I was kind of just following what was done in `include_attachments` (also, I'm not a huge fan of list comprehensions only because they're harder/less clear for me to read).

File: actions/lib/formatters.py
Comment (mjtice) [reply actions/lib/formatters.py]: I just mimicked the style of the other functions.  I'll leave it as-is just to be consistent if that's ok.

**Comment by mjtice** [comment]
I made some changes per the suggestions above.  If you're ok with them, let's merge this.

**Comment by mjtice** [comment]
Actually, hold tight on that merge.  I think there are some more things I'd like to add to this if that's ok.

File: actions/lib/formatters.py
Comment (floatingstatic) [reply actions/lib/formatters.py]: Not a hill i will die on, thats fine.

**Comment by floatingstatic** [comment]
@mjtice thats fine, just let us know when its 100% ready for final review, thanks!

**Comment by floatingstatic** [comment]
@mjtice Can you rebase and bump the pack version to 2.5.0. Sorry I ended up merging another PR which created a conflict. I think we can merge this after those changes and whenever you confirm you have made all changes for final review.

**Pull request closed without merging**
